.github/workflows/release.yml

```diff
@@ -10,33 +10,37 @@ jobs:
     name: Release
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
       - name: Parse release description
         id: split_description
         uses: jungwinter/split@v2
         with:
           msg: ${{ github.event.release.body }}
 
-      - name: Set versions
-        id: versions
+      - name: Set vars
```

Comment:
Should we update docs here?

Reply:
No, [docs ](https://github.com/dell/csi-baremetal-operator/blob/gh-pages/docs/index.yaml) are updated in operator repo: https://github.com/dell/csi-baremetal-operator/blob/b410ffb9cdaa48d8b1e680ff22d54036293c628d/.github/workflows/release.yml#L128

Reply:
I mean https://github.com/dell/csi-baremetal/blob/master/docs/release-workflow.md

Reply:
Seems like it is not related now and has no info about actions' usage

Reply:
Oh, I got it now, ok, I'll upgrade it, thanks.

Reply:
Doc is updated.

Reply:
thx!